### DISCUSSION TITLE
Rails 7 support

### DISCUSSION THREAD

**Discussion by mnifakram** [mongodb/mongoid#5107] [Q&A] [answered]
Is there any plan to support Rails 7?

```
Bundler could not find compatible versions for gem "activemodel":
  In Gemfile:
    mongoid was resolved to 7.3.0, which depends on
      activemodel (>= 6.0, < 6.2)

    rails (~> 7.0.0) was resolved to 7.0.0, which depends on
      activemodel (= 7.0.0)
```

**Comment by comandeo** [comment]
Hi @mnifakram, 
Unfortunately, we did not manage to add support for Rails 7 before its release. But we plan to do it early next year.
There is a ticket for this task in our backlog for this topic - https://jira.mongodb.org/browse/MONGOID-5193

**Comment by OskarEichler** [comment]
@comandeo Really looking forward to this - currently the dependencies falsely resolve this to mongoid (4.0.0.beta2).

Maybe good to take down this release or modify the gemspec until Rails 7 is officially supported!

**Reply by OskarEichler** [reply]
Here: https://github.com/mongodb/mongoid/tree/v4.0.0.beta2

**Comment by midnight-wonderer** [comment]
Wow, a major version bump.  
This will be released in Mongoid 8.0; I hope it won't take long.

**Comment by comandeo** [comment, marked as answer]
We have just released Mongoid 7.3.4 with Rails 7 support - https://github.com/mongodb/mongoid/releases/tag/v7.3.4 :tada:

**Reply by Lewiscowles1986** [reply]
Amazing!
